libfwupd/fwupd-enums.h

```diff
@@ -1339,6 +1339,14 @@ typedef enum {
 	 * Since: 1.9.24
 	 */
 	FWUPD_VERSION_FORMAT_DELL_BIOS_MSB,
+	/**
+	 * FWUPD_VERSION_FORMAT_INTEL_CSME19:
+	 *
+	 * Intel ME-style A.B.CC.DDDD notation notation.
```

Comment:
Your number example is AA not A

Reply:
Good catch, I've updated FWUPD_VERSION_FORMAT_INTEL_ME2 to make it clearer.